include/swift/AST/AutoDiff.h

```diff
@@ -202,9 +206,6 @@ class AutoDiffParameterIndicesBuilder {
   /// Sets the parameters at indices in the specified range.
   void setParameters(unsigned lowerBound, unsigned upperBound);
 
-  /// Sets all parameters.
-  void setAllParameters();
```

Comment:
Removed `setAllParameters` because it has no usages and is subsumed by `inferParameters`.

It would be possible to implement `inferParameters` using `setAllParameters` if `AutoDiffParameterIndicesBuilder` stored a `functionType`, but it doesn't.